tests/test_redis_cond_var.py

```diff
@@ -1,5 +1,7 @@
 import asyncio
-import aioredis
+# See https://github.com/aio-libs-abandoned/aioredis-py/issues/1443"
+from redis import asyncio as aioredis
+# import aioredis
```

Comment:
You need to update `requirements.txt` to support this change with:
* remove `aioredis`
* update `redis` requirement to `redis>=4.2.0rc1`